native/Avalonia.Native/src/OSX/window.mm

```diff
@@ -309,9 +318,10 @@ virtual HRESULT Resize(double x, double y) override
                 BaseEvents->Resized(AvnSize{x,y});
             }
             
-            [StandardContainer setFrameSize:NSSize{x,y}];
             [Window setContentSize:NSSize{x, y}];
             
+            _inResize = false;
```

Comment:
Do we need to worry about exceptions causing this not to be reset here?

Reply:
added a try - finally.